test/stripe/stripe_client_test.rb

```diff
@@ -244,7 +271,9 @@ class StripeClientTest < Test::Unit::TestCase
         assert_equal connection_manager_two.config.open_timeout, 100
       end
 
-      should "create a single connection manager for identitical configurations" do
+      should "create a single connection manager for identical configurations" do
+        StripeClient.clear_all_connection_managers
```

Comment:
Because connection managers are now cleared more granularly, it's possible to have some leftover from previous tests. Force clear them here so that we get the expected numbers below.

Reply:
Oh, my. I routinely put my English degree to shame. Sorry about that typo :smile:

Reply:
Haha, you'd need a pretty advanced degree to stop making typos ;)